api-contract/contract.yaml

```diff
@@ -654,6 +740,50 @@ components:
           type: string
         updatedAt:
           type: string
+    Notification:
+      type: object
+      properties:
+        liked:
+          type: string
+        reached100Likes:
+          type: string
+          items:
+            $ref: '#/components/schemas/Post'
+        reached25Friends:
+          type: string
+          items:
+            $ref: '#/components/schemas/UserProfile'
+        friendRequest:
+          type: string
+        requestAccepted:
+          type: string
+        commented:
+          type: string
+        userId:
+          type: string
+        notifId:
+          type: string
+        postID:
+          type: string
```

Comment:
We need to think about this one again. Basically the point of the notification is to have:
* an id, notifId iD is not a good name, we should use something more obvious, like notificationId (also to be consistent with the other schemas)
* we need to have IDs of the schemas that we provide notifications for, e.g. user / post / comment and so on. I would implement it by just adding the properties: userId, postId, commentId
* we need to have a message, literally a property just named: 'message'
* we need to have a boolean to know if the notification was read or not, another property named 'read'

I think that's it pretty much